lib/content-services/src/lib/services/alfresco-api.service.ts

```diff
@@ -122,15 +120,26 @@ export class AlfrescoApiService {
         if (this.alfrescoApiFactory) {
             return this.alfrescoApiFactory.createAlfrescoApi(config);
         }
-        return new AlfrescoApi(config);
+
+        return new AlfrescoApi(
+            {
+                ...config,
+                oauthInit: false
+            },
+            this.adfHttpClient
+        );
```

Comment:
The previous implementation was designed to maintain backward compatibility for consumers using the old solution with authentication provided via js-api. Including `this.adfHttpClient` here will enforce the new authentication method for all users.

the `AlfrescoApiNoAuthService` was used only when the consumers decide to use the`AuthModule`

Reply:
Please review PR in full... the Content Services module thinks different :)

```
{ provide: AlfrescoApiService, useClass: AlfrescoApiNoAuthService }
```

Literally, entire Content Services used `AlfrescoApiNoAuthService`